services/QuillLMS/spec/models/user_spec.rb

```diff
@@ -386,6 +386,25 @@
 
       it { expect { subject }.to change(user, :name).from(name).to('Test McTest') }
     end
+
+    context 'van is the first name' do
+      let(:name) { 'van nieman'}
+
+      it { expect { subject }.to change(user, :name).from(name).to('Van Nieman') }
+    end
+
+    context 'dit is the first name' do
+      let(:name) { 'dit ditsy'}
+
+      it { expect { subject }.to change(user, :name).from(name).to('Dit Ditsy') }
+    end
+
+    context 'van is in the middle' do
+      let(:name) { 'betsy van foo'}
+
+      it { expect { subject }.to change(user, :name).from(name).to('Betsy Van Foo') }
```

Comment:
Our system doesn't distinguish between middle and last names.  In all likelihood, this should most likely be 'Betsy van Foo'.